src/components/copyText/test/copyText.test.tsx

```diff
@@ -0,0 +1,50 @@
+import { act, fireEvent, render, screen } from '@testing-library/react';
+import CopyText, { copyToClipboard } from '../index';
+import React from 'react';
+
+Object.assign(navigator, {
```

Comment:
In this case, you made onCopy a function you can pass in, why not just pass in a jest.fn instead of mutating a global var? It makes the tests a lot more readable

Reply:
Ah I see nevermind, see the comment below

Reply:
Maybe a line describing what this is doing would help here